pages/index.tsx

```diff
@@ -285,13 +284,13 @@ const Dashboard: React.FC = () => {
 
       <div className="col-12 xl:col-6">
         <div className="card">
-          <h5>Sales Overview</h5>
-          <Chart type="line" data={lineData} options={lineOptions} />
+          <h5>All Open Cases</h5>
```

Comment:
I wonder if we should rename this, at least until I'm able to implement pagination

Reply:
Rename it to what? that was on Kate's Design... Technically the current table falls under this table view since other tables are closed cases, and my cases